.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "streetsidesoftware.code-spell-checker", //https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker
+    ]
+}
```

Comment:
Did you intend to commit this file?

Reply:
Yes, added explanation to the PR description.